src/utils.ts

```diff
@@ -233,3 +235,15 @@ export function getTouches(event: InputEventType, eventCache: PointerEvent[]) {
 	}
 	return 0; // case of MouseEvent
 }
+
+export function setCssProps(element: HTMLElement, originalCssProps?: { [key: string]: string; }): { [key: string]: string; } {
+	const oldCssProps = {};
```

Comment:
It seems to be possible to change and restore using cssText.

Reply:
In case of element having original inline css, or also having touch-action property on original inline css. Saving original style and also adding axes specific styles can be more complicated by using cssText. Therefore, element.style is used for saving original style and setting axes specific styles.